### ISSUE TITLE
Cannot assign accessibility properties to any elements

### ISSUE THREAD

**Issue by simonepizzamiglio** [Gil2015/react-native-table-component#108]
Hi,

I'm trying to assign accessibility properties such as accessibilityLabel and testID to both components of this library and custom components within the table component but they are not assigned.

How could I solve this problem? I'm not able to test my table.

**Comment by anabeatrizzz** [comment]
Same issue.
